src/sentry/api/endpoints/source_map_debug_blue_thunder_edition.py

```diff
@@ -0,0 +1,494 @@
+        organization_id=project.organization_id,
+        release_id=release.id,
+        name__in=possible_urls,
+    ).exclude(artifact_count=0)
+    if len(possible_release_files) > 0:
+        result = "wrong-dist"
+    for possible_release_file in possible_release_files:
+        if possible_release_file.ident == ReleaseFile.get_ident(
+            possible_release_file.name, event.dist
+        ):
+            return "found"
+    return result
+
+
+def get_artifact_bundle_file_status_by_url(event, project, release, possible_urls):
+    result = "unsuccessful"
+    possible_release_artifact_bundles = ReleaseArtifactBundle.objects.filter(
+        organization_id=project.organization.id,
+        release_name=release.version,
+        artifact_bundle__projectartifactbundle__project_id=project.id,
+        artifact_bundle__artifactbundleindex__organization_id=project.organization.id,
+        artifact_bundle__artifactbundleindex__url__in=possible_urls,
+    )
+    if len(possible_release_artifact_bundles) > 0:
+        result = "wrong-dist"
+    for possible_release_artifact_bundle in possible_release_artifact_bundles:
+        if possible_release_artifact_bundle.dist_name == (event.dist or ""):
+            return "found"
+    return result
+
+
+def get_matching_source_map_location(source_file_path, source_map_reference):
+    return non_standard_url_join(force_str(source_file_path), force_str(source_map_reference))
+
+
+def event_has_debug_ids(event_data):
+    debug_images = get_path(event_data, "debug_meta", "images")
+    if debug_images is None:
+        return False
+    else:
+        for debug_image in debug_images:
+            if debug_image["type"] == "sourcemap":
+                return True
+        return False
+
+
+def get_sdk_debug_id_support(event_data):
+    sdk_name = get_path(event_data, "sdk", "name")
+
+    official_sdks = None
+    try:
+        sdk_release_registry = get_sdk_index()
+        official_sdks = [
+            sdk.startswith("sentry.javascript.") for sdk in sdk_release_registry.keys()
+        ]
+    except Exception as e:
+        sentry_sdk.capture_exception(e)
+        pass
+
+    if official_sdks is None or len(official_sdks) == 0:
+        # Fallback list if release registry is not available
+        official_sdks = [
+            "sentry.javascript.angular",
+            "sentry.javascript.angular-ivy",
+            "sentry.javascript.browser",
+            "sentry.javascript.capacitor",
+            "sentry.javascript.cordova",
+            "sentry.javascript.electron",
+            "sentry.javascript.gatsby",
+            "sentry.javascript.nextjs",
+            "sentry.javascript.node",
+            "sentry.javascript.opentelemetry-node",
+            "sentry.javascript.react",
+            "sentry.javascript.react-native",
+            "sentry.javascript.remix",
+            "sentry.javascript.svelte",
+            "sentry.javascript.sveltekit",
+            "sentry.javascript.vue",
+        ]
```

Comment:
l: Given it's just a fallback, I think this is fine but have you thought about taking the list from here? 
 https://github.com/getsentry/sentry/blob/2833663bc83c9cef0f581958b4932ecf8a403de5/src/sentry/utils/platform_categories.py#L38-L49

Reply:
feel free to disgregard btw. Whenever I added new platforms, I searched for the last platform we added and would have found this list anyway.

Reply:
I think these are different identifiers. I will stick with this fallback for now. I don't want to overthink it.